Cargo.toml

```
@@ -1,26 +1,29 @@
 [package]
-name = "promscale_rs"
-version = "0.1.3"
-authors = ["Joshua Lockerman <josh@timescale.com>"]
+name = "promscale"
+version = "0.4.0"
 edition = "2018"
 
 [lib]
-crate-type = ["staticlib"]
+crate-type = ["cdylib", "rlib"]
 
 [profile.release]
 lto = true
 
+[profile.dev]
+lto = "thin"
+
```

The recommended config is something like

```suggestion
[profile.dev]
panic = "unwind"
lto = "thin"

[profile.release]
panic = "unwind"
opt-level = 3
lto = "fat"
debug = true
codegen-units = 1
```
basically:
1. want to ensure unwinding so we don't abort a postgres process
2. fat lto does a better job cleaning up unused symbols, and helps ensure that we don't leak extra ones into the binary (though it makes compiling slower)
3. leaving in debug info makes it easier to get info when things go wrong
4. everything else is just an optimization setting which tbh isn't important

---

Thanks, I have taken on these changes